Commonality Analysis Code_2.Rmd

```diff
@@ -0,0 +1,373 @@
+---
+title: "Commonality Analysis Code_2"
+author: "Laura Jans and Arielle Smith"
+date: "2023-07-25"
+output: html_document
+---
+
+# Loading Cleaned Data
+
+```{r setup, echo=TRUE, message=FALSE, warning=FALSE}
+
+#Load required packages
+
+if(!require(tidyverse)){install.packages('tidyverse')}
+library(tidyverse)
+
+if(!require(psych)){install.packages('psych')}
+library(psych)
+
+if(!require(Hmisc)){install.packages('Hmisc')}
+library(Hmisc)
+
+if(!require(yhat)){install.packages('yhat')}
+library(yhat)
+
+if(!require(boot)){install.packages('boot')}
+library(boot)
+
+if(!require(VennDiagram)){install.packages('VennDiagram')}
+library(VennDiagram)
+
+if(!require(gsheet)){install.packages('gsheet')}
+library('gsheet')
+
+if(!require(ggrepel)){install.packages('ggrepel')}
+library('ggrepel')
+
+if(!require(stargazer)){install.packages('stargazer')}
+library(stargazer)
+
+#Set working directory
+
+setwd("Z:/Commonality Analysis") # Change based on computer mapping
+
+#Read in data from data cleaning and pre-processing code (to avoid re-running data prep every time)
+
+cope_ca_data_imputed <- read.csv("cope_ca_data_imputed")
+cope_ca_data_nonimputed <- read.csv("cope_ca_data_nonimputed")
+
+```
+
+# Creating Residual Change Scores
+
+## Dependent Variables
+
+```{r, results = "asis", include=T, echo = T, message=FALSE, warning=FALSE}
+

```

Comment:
Again, as far as I know, everything should be imputed! You shouldn't have to do this here

Reply:
Since we did not impute missing baseline variables for SSI completers (N = 2, I think) I had originally filtered these cases out, since I could not make residuals with the missing baseline means.  However, we ended up adding these lines of code to the other file (before imputation -- as Arielle noted above).

Reply:
(So I also removed these lines from this file!)